src/config/kernel.md

```diff
@@ -3,16 +3,21 @@
 ## Kernel series
 
 Void Linux provides many kernel series in the default repository. These are
-named `linuxX.X`, for example `linux4.19`. You can query for all available
+named `linuxX.Y`: for example, `linux4.19`. You can query for all available
 kernel series by running:
 
 ```
 $ xbps-query --regex -Rs '^linux[0-9.]+-[0-9._]+'
 ```
 
-The `linux` meta package which is installed by default depends on one of the
-kernel packages, usually the latest mainline kernel that works with all dkms
-modules.
+The `linux` meta package, installed by default, depends on one of the kernel
+packages, usually the latest mainline kernel that works with all DKMS modules.
+Newer kernels might be available in the repository, but are not necessarily
+considered stable enough to be the default; use these at your own risk. If you
+wish to use a more recent kernel and have DKMS modules that you need to build,
+install the relevant `linuxX.Y-headers` package, then use
+[xbps-reconfigure(1)](https://man.voidlinux.org/xbps-reconfigure.1) to
+reconfigure the `linuxX.Y` package you installed.
```

Comment:
Make it clearer that the reconfiguration step will build the DKMS modules? Not sure if it's needed.

Reply:
i don't think it'll hurt, so i've just added "This will build the DKMS modules" to the end of that para.